shims/common/src/main/java/org/apache/hadoop/hive/thrift/HadoopThriftAuthBridge.java

```diff
@@ -326,6 +334,15 @@ public void setSecretManager(DelegationTokenSecretManager secretManager) {
       this.secretManager = secretManager;
     }
 
+    /*
+    public void setConf(Configuration conf) {
+      this.thriftConf = conf;
+    }
+    */
+    public void setClassName(String customClassName) {
```

Comment:
I would strongly suggest to use a different method name here, e.g. setCustomAuthClassName?